unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +49,10 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist)
+{
+    std::unordered_multiset<Song> first_playlistset(first_playlist.begin(),first_playlist.end());
```

Comment:
also you could check the size before the copying

Reply:
okk